### PR TITLE
wrap examples for easier display

### PR DESCRIPTION
**Pull request by dasormeter**
Add linebreaks in readme examples to make more readable

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/unreleased.rst), if appropriate
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: BAZEL.md [hunk 1]
```diff
@@ -370,13 +370,18 @@ detailed information.
 - Execute a test with a specific name
 
     ```
-    bazel test //ledger/sandbox:sandbox-scala-tests_test_suite_src_test_suite_scala_com_digitalasset_platform_sandbox_stores_ledger_sql_JdbcLedgerDaoSpec.scala --test_arg=-t --test_arg="JDBC Ledger DAO should be able to persist and load contracts without external offset"
-    ```    
+    bazel test 
+    //ledger/sandbox:sandbox-scala-tests_test_suite_src_test_suite_scala_com_digitalasset_platform_sandbox_stores_ledger_sql_JdbcLedgerDaoSpec.scala 
+    --test_arg=-t 
+    --test_arg="JDBC Ledger DAO should be able to persist and load contracts without external offset"
+    ```
```
Comment (SamirTalwar) [inline BAZEL.md]: Perhaps add ` \` to the end of each line (aside from the last one) to signal continuation?

Comment (aherrmann-da) [reply BAZEL.md]: Agreed, making the continuations more obvious would be good. `\` line endings and maybe also indenting the continued lines a bit.

Comment (dasormeter) [reply BAZEL.md]: updated to include the slashes

File: BAZEL.md [hunk 2]
```diff
@@ -370,13 +382,18 @@ detailed information.
 - Execute a test with a specific name
 
     ```
-    bazel test //ledger/sandbox:sandbox-scala-tests_test_suite_src_test_suite_scala_com_digitalasset_platform_sandbox_stores_ledger_sql_JdbcLedgerDaoSpec.scala --test_arg=-t --test_arg="JDBC Ledger DAO should be able to persist and load contracts without external offset"
-    ```    
+    bazel test \
+    //ledger/sandbox:sandbox-scala-tests_test_suite_src_test_suite_scala_com_digitalasset_platform_sandbox_stores_ledger_sql_JdbcLedgerDaoSpec.scala \
+    --test_arg=-t \
+    --test_arg="JDBC Ledger DAO should be able to persist and load contracts without external offset"
```
Comment (SamirTalwar) [inline BAZEL.md]: I'm with @aherrmann-da in that it'd be nice to have the continued lines indented by 2 spaces, but mostly I'm happy.

**Pull request merged**